pkg/controller/pd_control.go

```diff
@@ -32,6 +32,34 @@ func GetPDClient(pdControl pdapi.PDControlInterface, tc *v1alpha1.TidbCluster) p
 	return pdControl.GetPDClient(pdapi.Namespace(tc.GetNamespace()), tc.GetName(), tc.IsTLSClusterEnabled())
 }
 
+// GetPDClient tries to return an available PDClient
+// If the pdClient built from the PD service name is unavailable, try to
+// build another one with the ClientURL in the PeerMembers.
+// ClientURL example:
+// ClientURL: https://cluster2-pd-0.cluster2-pd-peer.pingcap.svc.cluster2.local
```

Comment:
```suggestion
// ClientURL example: https://cluster2-pd-0.cluster2-pd-peer.pingcap.svc.cluster2.local
```

Reply:
We can provide more examples in the future if needed. There's no need to merge these lines.